about.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About - Melissa Jimison</title>
+  <meta charset="UTF-8">
+  <link href='https://fonts.googleapis.com/css?family=Shadows+Into+Light' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" type="css" href="stylesheets/styles.css">
+</head>
+<body>
+
+  <header class="header-about">
+    <nav>
+      <ul>
+        <a class="logo" href="file:///Users/melissajimison/Documents/c5/projects/Static-Site/portfolio.html">Melissa Jimison</a>
+        <li><a href="file:///Users/melissajimison/Documents/c5/projects/Static-Site/portfolio.html">Ruby Portfolio</a></li>
+        <li><a href="file:///Users/melissajimison/Documents/c5/projects/Static-Site/code-journal.html">Code Journal</a></li>
+        <li><a href="file:///Users/melissajimison/Documents/c5/projects/Static-Site/about.html">About</a></li>
+      </ul>
+    </nav>
+
+    <div class="intro-journal">
+      <h1 class="pages-tittle"> About me </h1>
+      <h2 class="pages-second-tittle">Former Colombian journalist and aspiring programmer</h2>
+    </div>
+  </header>
+
+  <section class="main-container" >
+    <article class="about-page">
+      <figure>
+        <img src="file:///Users/melissajimison/Documents/c5/projects/Static-Site/images/about.jpg" alt="Melissa Jimison" />
+      </figure>
+      <h4>Melissa Jimison</h4>
+      <h5>Ada Student</h5>
+      <h6>About</h6>
+      <p>Former Colombian journalist aspiring programmer and universe conquer. A lover of Ruby for now. In the past two year cultural analizer full time. Specialist in blowing people's mind and in 'anintentional' ratial comments </p>
+    </article>
+
+    </section>
+
+    <footer>
+
+      <figure class="social-media-box">
+        <a href="https://www.facebook.com/MelissaJimison"> <img class="social-media" src="file:///Users/melissajimison/Documents/c5/projects/Static-Site/images/facebook-logo.jpg" alt="Facebook logo" target="_blank">
+        </a>
+
+        <a href="https://twitter.com/melissajimison"> <img class="social-media" src="file:///Users/melissajimison/Documents/c5/projects/Static-Site/images/twitter-logo.jpg" alt="Twitter logo" target="_blank">
+        </a>
+
+        <a href="https://instagram.com/melissajimison"> <img class="social-media" src="file:///Users/melissajimison/Documents/c5/projects/Static-Site/images/instagram-logo.jpg" alt="Instagram logo" target="_blank">
+        </a>
+
+        <a href="hhttps://co.linkedin.com/in/melijimison"> <img class="social-media" src="file:///Users/melissajimison/Documents/c5/projects/Static-Site/images/linkedin-logo.jpg" alt="Linkedin logo" target="_blank">
+        </a>
+      </figure>
+
+      <copy-rights>
```

Comment:
`<copy-rights>` isn't a valid html element. `<small>` is commonly used for copyright info, or you could use `<p>`.